apps/bot-discord/src/config.json

```diff
@@ -42,16 +42,12 @@
     "channelId": "1135563848586379264",
     "stickyMessage": {
       "title": "Sticky notice",
-      "description": "In this channel, you can get assistance with your questions, issues, or concerns. Before continuing, please continue to read this sticky message.",
+      "description": "Read the pinned messages and <#953993848374325269> for common problems.",
```

Comment:
We may leave the description empty and add a new field `🔸 Before asking for assistance`

Reply:
Then why did you keep the other field then?

Reply:
Not sure which one you mean? There should only be one field in the embed I'm proposing.

Reply:
> No that increases the sticky notice by one line without providing any actual information. Any additional clutter must be cut down to raise the likelihood of it actually being read by users

Reply:
If you're talking about "Properly ask for assistance". This is partially in the interest of semantics (having a header for the list) and partially because it does provide context as to why the list even exists. Additionally it adds some visual separation between what the user should do before asking and how to ask.

Reply:
> interest of semantics (having a header for the list) 

Why does a list require a field? I can simply put it in the description using your previous argumentation. 

> because it does provide context as to why the list even exists

The same way `🔸 Before asking for assistance` would provide context why `Read the pinned messages and <#953993848374325269> for common problems.` exists. The context would be `🔸 Before asking for assistance` and the answer would be `Read the pinned messages and <#953993848374325269> for common problems.`.

>  what the user should do before asking

This would be formalized in the field header as `🔸 Before asking for assistance`. As of now there is no such thing as "what the user should do before asking" like you say.

Reply:
>Why does a list require a field? I can simply put it in the description using your previous argumentation.

From my understanding, because the field causes a paragraph break which is simply more legible.

>The same way 🔸 Before asking for assistance would provide context

Correct but given that one should read that action anyways, it does not really require any context as the sentence is clear as day by itself. In comparison, the list by itself would not be particularly clear as it does not include any call to action. "Read the pinned messages" does not require a call to action, as it already is a call to action.

>As of now there is no such thing as "what the user should do before asking" like you say.

Same thing as above, we don't need to give the users two instructions here. 


Additionally, I might add that the sticky message only contains two parts. Inducing separation between two parts does not require two segments.

Reply:
> From my understanding, because the field causes a paragraph break which is simply more legible.

A paragraph break can be added to the description too? So why not move it to a description then? Because otherwise:

> [the field] increases the sticky notice by one line without providing any actual information

> "Read the pinned messages" does not require a call to action, as it already is a call to action.

You should do it before asking for assistance. This information is missing. It can be added to the description or as a field header. If it is added to the description, then the same way I can remove the field `🔸 Properly ask for assistance` and instead add it to the description "To properly ask for assistance include the following..." like you had your field header originally. So why do you keep it as a field then?

> Additionally, I might add that the sticky message only contains two parts. Inducing separation between two parts does not require two segments.

This is wrong semantics. The description is global to the entire embed, whereas the fields are specific sections under the description. Before this PR, the description acted globally as an introduction to the channel and the fields went into specific topics such as how to ask for assistance or what to do before creating perfect semantic, now, you get rid of what is supposed to be a field and cram it into the description when the description is not meant to be used that way. If you want to use the description to include specific information, you would have to remove the current field. If you map this object oriented you have two "CallToAction" objects. One "What to do before asking" and one "How to ask" "CallToAction" object. They are both of type "CallToAction" but you treat one differently than the other by including it in the description instead of the field, creating a semantical difference between these objects which do not exist.

Reply:
>A paragraph break can be added to the description too?

Not entirely sure, as \n is just a line break but sure if its possible?

>If it is added to the description, then the same way I can remove the field 🔸 Properly ask for assistance and instead add it to the description 

Correct but it would make the description appear as too big of a text block which drastically decreases the chance of it actually being read. Same principle as to why I made an item list in the first place.

>Before this PR, the description acted globally as an introduction to the channel
>and cram it into the description when the description is not meant to be used that way

I am doing so because it avoids an additional line of text where it is not needed.
If possible I'd remove the description text and add another field, but not add title for that field.

However I wasn't sure if a field with no header would work on a technical basis. If it does, we can remove the description and replace it with a field with no header. Alternatively one might also have the text be in the header, instead of the field content.

>but you treat one differently than the other by including it

Correct because there is no need to tell the user to read the pins before asking. They should read the pins at any stage. The call to action of "read" is in the sentence itself.
The list itself does not provide such call to action by itself, hence why it needs a header.